src/js/App/FeatureFlags/FeatureFlagsProvider.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UnleasFlagProvider from '@unleash/proxy-client-react';
+
+const config = {
+  url: `${document.location.origin}/api/featureflags/v0`,
+  clientKey: 'proxy-123',
```

Comment:
Shouldn't the `clientKey` be changable? Otherwise it looks like a magic value.

Reply:
We have a custom unleash proxy backed that is not using the key, but the client requires it. I think eventually the client key will be our user number. But we are not there yet.